packages/Menu/src/Menu.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState, useCallback, EventHandler } from 'react';
+import PropTypes from 'prop-types';
+import Popover, { Align, Justify, PopoverProps } from '@leafygreen-ui/popover';
+import { useDocumentEventListener } from './hooks';
+import { emotion } from '@leafygreen-ui/lib';
+import { colors } from '@leafygreen-ui/theme';
+import { cx } from 'emotion';
+
+const { css } = emotion;
+
+const rootMenuStyle = css`
+  width: 200px;
+  border-radius: 3px;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.15);
+  background-color: ${colors.mongodb.white};
+`;
+
+interface MenuProps extends Omit<PopoverProps, 'spacing'> {
```

Comment:
I _think_ we can do `Omit<PopoverProps, PopoverProps.spacing>`.

Reply:
Does not pass my TS linter that way ^ @DesignerDave

Reply:
Then ignore this comment 😁